build/common.go

```diff
@@ -99,7 +99,12 @@ func buildBackend(cfg Config) error {
 	}
 
 	// TODO: Change to sh.RunWithV once available.
-	return sh.RunWith(cfg.Env, "go", args...)
+	err = sh.RunWith(cfg.Env, "go", args...)
```

Comment:
Nit, unrelated: the comment above specifies "TODO: Change to sh.RunWithV once available.", it is now available, maybe we can kill two birds with one stone? :slightly_smiling_face: 

```suggestion
	err = sh.RunWithV(cfg.Env, "go", args...)
```